templates/components/password.build.hbs

```diff
@@ -1,6 +1,9 @@
 <input
   type="password"
   class="form-control"
+  :class="{ disabled: autogenerate }"
+  :readonly="autogenerate"
+  autocomplete="off"
   v-model.trim="value"
```

Comment:
I assume spaces are allowed in passwords, so we shouldn't be trimming the value?

Reply:
I didn't change this but I'm OK in getting rid of it.

Reply:
I know. I just spotted it. Let's get rid of the trim please.